deploy/base/config.yaml

```diff
@@ -7,3 +7,13 @@ metadata:
 spec:
   enableSelinux: false
   enableLogEnricher: false
+  tolerations:
+    - key: "node-role.kubernetes.io/master"
+      operator: "Exists"
+      effect: "NoSchedule"
+    - key: "node-role.kubernetes.io/control-plane"
+      operator: "Exists"
+      effect: "NoSchedule"
+    - key: "node.kubernetes.io/not-ready"
+      operator: "Exists"
+      effect: "NoExecute"
```

Comment:
let's remove the `not-ready` toleration. The rest of them are fine. If the kubelet is stating that it's not ready to schedule workloads, let's respect that. The SPO is not considered a critical part of Kube (yet :smiley: )